mcs/class/referencesource/mscorlib/system/security/cryptography/rc2cryptoserviceprovider.cs

```diff
@@ -63,7 +63,12 @@ public sealed class RC2CryptoServiceProvider : RC2 {
                 return m_use40bitSalt;
             }
             set {
+#if MONO
```

Comment:
This is unnecessary, if you want to add such check add it to the place where the value is used and it's not false